lib/apiservers/engine/backends/system.go

```diff
@@ -407,7 +407,7 @@ func (s *SystemBackend) AuthenticateToRegistry(ctx context.Context, authConfig *
 		log.Debugf("logging onto %s", authURL.String())
```

Comment:
This and a couple of other `log.XXX` calls should be updated to use the `op.XXX` equivalent.